src/BlazorUI/Bit.BlazorUI/Components/Navs/DropMenu/BitDropMenu.scss

```
@@ -77,61 +153,121 @@
+    --bit-drm-cal-opa: 0;
+    --bit-drm-cal-tra: var(--bit-drm-cal-tra-ent);
+
     display: none;
+    outline: transparent;
     position: fixed;
     box-sizing: border-box;
     z-index: $zindex-callout;
-    animation-fill-mode: both;
-    animation-duration: $mot-duration-long;
-    background-color: $clr-bg-pri;
+    // The sizing the consumer asked for, defaulted to what the callout does on its own: as wide as its
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)